_includes/code/howto/java/src/test/java/io/weaviate/docs/manage-data.classes.java

```diff
@@ -72,6 +72,34 @@ private void createCollection(String className) {
       .returns(true, Result::getResult);
   }
 
+  private void createCollectionWithProperties(String className){
+    // START CreateCollectionWithProperties"
+    String className = className;
+    
+    Property titleProperty = Property.builder()
+              .name("title")
+              .description("Title Property Description...")
+              .dataType(Arrays.asList(DataType.TEXT))
+              .build();
+    
+    Property bodyProperty = Property.builder()
+              .name("body")
+              .description("Body Property Description...")
+              .dataType(Arrays.asList(DataType.TEXT))
+              .build();
+    
+    WeaviateClass articleClass = WeaviateClass.builder()
```

Comment:
It looks like java works a little differently than Python and TypeScript.

Do you need to define the `titleProperty` and `bodyProperty` before you add them to the `articleClass` definition? If so, consider adding a comment on line 78 "Define class properties" and a 2d comment on line 93 "Add the defined properties to the class"